ci/jenkins_tests/perf_integration_tests/run_perf_integration.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Show explicitly which commands are currently running.
+set -ex
+
+python -m pip install pytest-benchmark
+
+python -m pytest --benchmark-autosave python/ray/tests/test_perf_integration.py
```

Comment:
These scripts should be made so they can run from any directory (e.g., whether you are in `ray/` or `ray/ci`, or wherever). You can see how this is done in https://github.com/ray-project/ray/blob/master/ci/long_running_tests/start_workloads.sh for example.

Reply:
Same with the next script